### ISSUE TITLE
[Bug] parallel download from multiple sources

### ISSUE THREAD

**Issue by yohanim** [tachiyomiorg/tachiyomi#3922]
## Device information

App version: 0.10.5-2763 (standard, 4cbe4977, 51)
Android version: 10 (SDK 29)
Android build ID: BLA-L29 10.0.0.159(C432E5R1P3)
Device brand: HUAWEI
Device manufacturer: HUAWEI
Device name: HWBLA
Device model: BLA-L29
Device product name: BLA-L29

## Steps to reproduce
1. Have multiple manga from more than 5 different sources (at least 6)
2. Have the auto download new updates checked
3. Check update

## Issue/Request
# expected
Parallel download from x (seems to be 5) chapters in parallel from different sources
Once a source is finished, launch the next chapter from another source to download

# actual
All chapters are in the download queue but the ones from 6th and more sources does not start download
We have to pause-resume download for them to continue

## Other details
Additional details and attachments.

**Comment by arkon** [comment]
6 is just as arbitrary as 5.

**Comment by KevinGossentCap** [comment]
My remark on the number (6) is that I don't see the bug if downloads of less than 6 sources are triggered.
The number to do in parallel is not that important if it goes to the next sources afterwards ;)

**Comment by jhonnysn** [comment]
I believe that it would be good for each download to be independent and could have a menu identifying how many threads would run in parallel, so more robust cell phones can put several and they would work on top of each different host without interfering with the others.

**Comment by arkon** [comment]
Duplicate of #5639
